README.md

```diff
@@ -25,37 +25,40 @@
 <br />
 
 ## Install Giskard 🐢
-You can install the latest version of Giskard from PyPi using pip :
+Install the latest version of Giskard from PyPi:
 ```sh
-pip install giskard -U
+pip install "giskard[llm]"
```

Comment:
It’s nice to add the dependencies required for the ‘llm’ feature.
If you want to underline that you are installing the latest version (as you did in Line 28) I would add the flag -U making the line:
pip install "giskard[llm]" -U